src/components/collection/SimilarCollections.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useTranslation } from 'react-i18next';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import { makeStyles } from '@material-ui/core/styles';
+import SimilarCollection from './SimilarCollection';
+import SimilarCollectionsHeader from './SimilarCollectionsHeader';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    flexGrow: 1,
+  },
+  paper: {
+    padding: theme.spacing(2),
+    textAlign: 'center',
+    color: theme.palette.text.secondary,
+  },
+}));
+
+function SimilarCollections({ similarCollections }) {
+  const { t } = useTranslation();
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      <SimilarCollectionsHeader />
+      {!similarCollections.length ? (
+        <div className="Main">
+          <Typography variant="h5" color="inherit">
+            {t('This collection is empty.')}
```

Comment:
You would want to change the text to say something like "No similar collections available."